ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -333,6 +333,7 @@ CO2C:
   opt_uses:
   - high_zone_air_co2_concentration_alarm
   - failed_zone_air_co2_concentration_alarm
+  - zone_air_co2_concentration_deadband_setpoint
```

Comment:
This needs to be added as a telemetry field. Please change to `zone_air_deadband_co2_concentration_setpoint`

Reply:
updated

Reply:
still needs to be added to telemetry fields

Reply:
@shambergoldstein - apologies forgot to commit telemetry yaml, updated now